app/src/main/res/layout/fragment_saving_account_details.xml

```diff
@@ -408,24 +405,24 @@
             </android.support.v7.widget.CardView>
 
             <LinearLayout
+                android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_margin="@dimen/default_padding"
-                android:layout_width="wrap_content"
                 android:orientation="horizontal">
 
                 <TextView
-                    android:layout_height="wrap_content"
                     android:layout_width="wrap_content"
-                    android:text="@string/need_help"/>
+                    android:layout_height="wrap_content"
+                    android:text="@string/need_help" />
 
                 <TextView
-                    android:clickable="true"
-                    android:focusable="true"
                     android:id="@+id/tv_help_line_number"
-                    android:layout_height="wrap_content"
                     android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:clickable="true"
+                    android:focusable="true"
                     android:text="@string/help_line_number"
-                    android:textColor="@color/blue"/>
+                    android:textColor="@color/blue" />
```

Comment:
@sashank27 Why you have added this space?

Reply:
@luckyman20 these all spacing came after I did ./gradlew check and ./gradlew build. Will edit these and send new PR.

Reply:
@sashank27 No need for making a new PR, just make the changes and commit them

Reply:
ok